.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-  - "8"
+  - "10"
```

Comment:
Should be be using `"lts/*"`, either instead or in addition?

Reply:
How about adding `lts/*`, since it's basically free, and will only cause more work for us when there is an actual issue?

Reply:
Uh, actually, adding `node` is probably better.
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions

Reply:
Done at https://github.com/cubing/icons/pull/43 !